src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -50,28 +48,36 @@ namespace Stratis.Bitcoin.Features.Miner
     /// </remarks>
     public class PosMinting
     {
-        public class StakeOutput
-        {
-            public UtxoStakeDescription UtxoStakeDescription;
-            public int Depth;
-        }
-
+        /// <summary>
+        /// Information related to UTXO that is required for staking.
+        /// </summary>
         public class UtxoStakeDescription
         {
-            public uint256 HashBlock;
-            public TxOut TxOut;
-            public OutPoint OutPoint;
-            public int OutputIndex;
-            public HdAddress Address;
-            public UnspentOutputs UtxoSet;
-            public WalletSecret Secret;
-            public Key Key;
+            /// <summary>Block's hash.</summary>
+            public uint256 HashBlock { get; set; }
+            /// <summary>UTXO.</summary>
+            public TxOut TxOut { get; set; }
+            /// <summary>Information about transaction id and index.</summary>
+            public OutPoint OutPoint { get; set; }
+            /// <summary>Transaction's address.</summary>
+            public HdAddress Address { get; set; }
+            /// <summary>Utxo set.</summary>
+            public UnspentOutputs UtxoSet { get; set; }
+            /// <summary><see cref="WalletSecret"/></summary>
```

Comment:
this is not good summary

Reply:
how about "Credentials to wallet that contains the private key for the staking UTXO."